### PR TITLE
FIL streamz notebook example

### PR DESCRIPTION
**Pull request by brhodes10**
* Adding notebook example for FIL and streamz pipeline

### CODE REVIEW

**Comment by GPUtester** [comment]
Please update the changelog in order to start CI tests.

View the gpuCI docs [here](https://docs.rapids.ai/gpuci#open-pr-or-update-pr).

**Comment by brhodes10** [comment]
rerun tests

File: examples/FIL_streamz_pipeline.ipynb [hunk 1]
```diff
@@ -0,0 +1,375 @@
+{
```
Comment (BartleyR) [inline examples/FIL_streamz_pipeline.ipynb:1]: <p>I know you link to the data, but do you want to include how to download that data and get it into Kafka? Perhaps that outside the scope of this notebook though.</p><p> </p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/rapidsai/clx/pull/273/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='0'/>

Comment (BartleyR) [inline examples/FIL_streamz_pipeline.ipynb:1]: <p>Aren't we publishing the model? </p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/rapidsai/clx/pull/273/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='6'/>

Comment (BartleyR) [inline examples/FIL_streamz_pipeline.ipynb:1]: <p>We need some more text about what this block of code is doing.</p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/rapidsai/clx/pull/273/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='19'/>

Comment (BartleyR) [inline examples/FIL_streamz_pipeline.ipynb:1]: <p>Same here. Let's expand a bit on what we're doing with this function.</p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/rapidsai/clx/pull/273/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='20'/>

Comment (BartleyR) [inline examples/FIL_streamz_pipeline.ipynb:1]: <p>Provide some text that summarizes what this function does.</p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/rapidsai/clx/pull/273/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='23'/>

Comment (BartleyR) [inline examples/FIL_streamz_pipeline.ipynb:1]: <p>It's a good idea to include some conclusions here and possible ideas for how things can be expanded in the future.</p><p> </p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/rapidsai/clx/pull/273/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='25'/>

**Review by raykallen — CHANGES_REQUESTED** [review]
Can you add in the notebook outputs as examples?

**Comment by brhodes10** [comment]
rerun tests

**Comment by brhodes10** [comment]
> Can you add in the notebook outputs as examples?

Thanks ! Updated

**Review by BartleyR — APPROVED** [review]

**Review by raykallen — APPROVED** [review]

**Pull request merged**